src/anm/analytics.js

```diff
@@ -33,40 +43,41 @@ var Analytics = function () {
     this.trackPlayingComplete = this.trackPlayer('playing_complete');
 };
 
-Analytics.prototype.track = function track(name, opts) {
-    opts = opts || {};
-    opts.name = name;
-    opts.referer = document.referrer;
-    opts.lang = navigator.language || navigator.userLanguage;
-    opts.url = location.href;
-    opts.screenHeight = screen.height;
-    opts.screenWidth = screen.width;
-    opts.windowHeight = window.innerHeight;
-    opts.windowWidth = window.innerWidth;
-    opts.timestamp = new Date().getTime();
+Analytics.prototype.track = function track(name, player, action) {
     if (this.enabled) {
-        this.queue.push(opts);
+        this.queue[player.viewId] = this.queue[player.viewId] || {
+                viewId: player.viewId,
+                projectId: player.anim.meta._anm_id,
+                referer: document.referrer,
+                lang: navigator.language || navigator.userLanguage,
+                url: location.href,
+                screenHeight: screen.height,
+                screenWidth: screen.width,
+                windowHeight: window.innerHeight,
+                windowWidth: window.innerWidth,
```

Comment:
yes I know. I causes relayout but it doesn't mean that it is slow(take a few ms, if not less). Because it happens only once, it's not in a loop or smth.